versions.yaml

```diff
@@ -1,4 +1,4 @@
-api: 0.10.0
-console: 0.11.0
+api: wildcard-cert
```

Comment:
Do these need to be changed back?

Reply:
After api and console is merged I will change it.